client/src/api/index.js

```diff
@@ -147,6 +148,7 @@ export default {
       return [tokenRequest, userRequest]
     },
     create: async (orcid, accessToken, refreshToken, email, grants) => {
+      console.log(orcid, accessToken, refreshToken, email, grants)
```

Comment:
I think we should start removing console.logs at this point - I'm going to be turning on the lint rule soon